scripts/aks-with-registry.sh

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit # exit immediately if a command exits with a non-zero status.
+set -o nounset # exit when script tries to use undeclared variables.
+set -o pipefail # make the pipeline fail if any command in it fails.
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# shellcheck source=hack/ensure-azcli.sh
+source "${REPO_ROOT}/hack/ensure-azcli.sh" # install az cli and login using WI
+# shellcheck source=hack/ensure-tags.sh
+source "${REPO_ROOT}/hack/ensure-tags.sh" # set the right timestamp and job name
+
+
+main() {
+  # aks defaults
+  export MGMT_CLUSTER_NAME="${MGMT_CLUSTER_NAME:-aks-capz}-${RANDOM_SUFFIX}" # management cluster name
+  export AKS_RESOURCE_GROUP="${AKS_RESOURCE_GROUP:-aks-capz}-${RANDOM_SUFFIX}" # resource group name
+  export AKS_NODE_RESOURCE_GROUP="node-${AKS_RESOURCE_GROUP}"
+  export KUBERNETES_VERSION="${KUBERNETES_VERSION:-v1.30.2}"
+  export AZURE_LOCATION="${AZURE_LOCATION:-westus2}"
+  export AKS_NODE_VM_SIZE="${AKS_NODE_VM_SIZE:-"Standard_B2s"}"
+  export AKS_NODE_COUNT="${AKS_NODE_COUNT:-1}"
+  export MGMT_CLUSTER_KUBECONFIG="${MGMT_CLUSTER_KUBECONFIG:-$REPO_ROOT/aks-mgmt.config}"
```

Comment:
let's add this new local kubeconfig filepath default to .gitignore (looks like the default identity id filepath is already there)

Reply:
Updated in the recent push